### PR TITLE
fix: GetPermissionsForUesr retrieves empty policy

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

Fix: #767

### CODE REVIEW

**Comment by hsluoyz** [comment]
@sagarshah1983 plz review.

**Comment by sagarshah1983** [comment]
Thank you for getting this fixed.

**Review by sagarshah1983 — APPROVED** [review]
Thanks @closetool / @hsluoyz

File: rbac_api.go [hunk 1]
```diff
@@ -15,6 +15,7 @@
 package casbin
 
 import (
+	"fmt"
```
Comment (hsluoyz) [inline rbac_api.go:18]: empty line

**Review by hsluoyz — CHANGES_REQUESTED** [review]
retrives -> retrieves

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**